ikasaneip/visualisation-dashboard/pom.xml

```diff
@@ -8,8 +8,8 @@
         <version>2.2.0-SNAPSHOT</version>
     </parent>
 
-    <artifactId>visualisation-dashboard</artifactId>
-    <name>Visualisation Dashboard</name>
+    <artifactId>ikasan-dashboard</artifactId>
```

Comment:
Isnt that going to clash with old dashboard ?

Reply:
There is no more (old) ikasan-dashboard as of 2.2.0. Module is still there, but not built. Module should also be removed before cut.